minigalaxy/constants.py

```diff
@@ -23,6 +23,7 @@
     ["es", _("Spanish")],
     ["sv", _("Swedish")],
     ["tr", _("Turkish")],
+   
```

Comment:
This empty new empty line is causing an issue with the linting test. Could you remove it?

Reply:
yeah thanks i am at work r8 now going to correct it when i get home :)